src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -96,6 +97,10 @@ public function onKernelResponse(ResponseEvent $event): void
 
         // do not capture redirects or modify XML HTTP Requests
         if ($request->isXmlHttpRequest()) {
+            if (self::ENABLED === $this->mode && $this->ajaxReplace) {
+                $response->headers->set('Symfony-Debug-Toolbar-Replace', '1');
```

Comment:
shouldn't we set this only when it's not already set? that'd allow opting out when the config is enabled

Reply:
True, will patch that later 👍

Reply:
done :heavy_check_mark: